Source/Core/VideoBackends/Software/TransformUnit.cpp

```diff
@@ -319,7 +319,7 @@ static void LightAlpha(const Vec3& pos, const Vec3& normal, u8 lightNum, const L
 
 void TransformColor(const InputVertexData* src, OutputVertexData* dst)
 {
-  for (u32 chan = 0; chan < xfmem.numChan.numColorChans; chan++)
+  for (u32 chan = 0; chan < 2; chan++)
```

Comment:
Would it be worthwhile making this a named constant (here and in the other file)? Pretty please, with a comment on top.